packager/media/formats/mp4/mp4_init_muxer.cc

```diff
@@ -0,0 +1,34 @@
+// Copyright 2014 Google LLC. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+#include <packager/media/formats/mp4/mp4_init_muxer.h>
+
+#include <absl/log/check.h>
+
+#include <packager/macros/logging.h>
+#include <packager/macros/status.h>
+
+namespace shaka {
+namespace media {
+namespace mp4 {
+
+MP4InitMuxer::MP4InitMuxer(const MuxerOptions& options) : MP4Muxer(options) {}
+
+MP4InitMuxer::~MP4InitMuxer() {}
+
+Status MP4InitMuxer::Finalize() {
+  if (!segmenter_) {
```

Comment:
It looks like neither `segmenter_` nor `to_be_initialized_` is needed by this class?  I think we can simplify this further by removing them here and reverting them in `MP4Muxer`.